src/Microsoft.Health.Dicom.Core/Features/Store/StoreDatasetValidator.cs

```diff
@@ -32,7 +33,7 @@ public class StoreDatasetValidator : IStoreDatasetValidator
     private readonly IQueryTagService _queryTagService;
     private readonly StoreMeter _storeMeter;
     private readonly IDicomRequestContextAccessor _dicomRequestContextAccessor;
-
+    private readonly ILogger _logger;
 
     public static readonly HashSet<DicomTag> RequiredCoreTags = new HashSet<DicomTag>()
```

Comment:
If this is `private`, it can be a mutable type and there is an oh-so minimal performance increase by not using an `interface`. But if it's `public`, we definitely want it to be some sort of immutable type. I like using a method though